Packs/CommonScripts/Scripts/EditServerConfig/EditServerConfig.py

```diff
@@ -0,0 +1,68 @@
+from CommonServerPython import *
+
+from typing import Dict, Any
+import traceback
+
+SERVER_SYSTEM_CONFIG_PATH = '/system/config'
+''' STANDALONE FUNCTION '''
+
+
+def get_current_server_config() -> dict:
+    res = execute_command("demisto-api-get", {"uri": SERVER_SYSTEM_CONFIG_PATH})
+    config_json = res['response']
+    return config_json.get('sysConf', {})
+
+
+def set_system_config(server_config: dict):
+    execute_command(
+        "demisto-api-post",
+        {
+            "uri": SERVER_SYSTEM_CONFIG_PATH,
+            "body": {"data": server_config,
+                     "version": -1},
+        })
+
+
+def remove_server_config(key: str, server_config: dict):
+    if key in server_config:
+        server_config.pop(key)
+        set_system_config(server_config)
+
+
+def update_server_config(key: str, value: str, server_config: dict):
+    if not value:
+        raise DemistoException(
+            "EditServerConfig Error: You must give a value when you want to update a server configuration.")
+    server_config[key] = value
```

Comment:
Should the updated values always be of type string?

Reply:
Yes, the server does the converting from string to the correct type in the background. For example (I gave a string and the value set is an int):
The task:
![image](https://user-images.githubusercontent.com/71635916/136755039-89ae1ca7-0d72-42bf-ab15-08c90267593e.png)
Output:
![image](https://user-images.githubusercontent.com/71635916/136755105-076a0ac5-a1cd-480a-a1e7-a47010a2fd43.png)
The result in the config:
![image](https://user-images.githubusercontent.com/71635916/136755150-050e2bd4-9687-42e3-be07-70595ef15988.png)